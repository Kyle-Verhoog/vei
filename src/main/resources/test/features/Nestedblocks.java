public class Nestedblocks {
  public Nestedblocks() {}
  public int m() {
    int x = 42;
    int y = 87;
    { int z = x+y;
      return z;
    }
  }
}