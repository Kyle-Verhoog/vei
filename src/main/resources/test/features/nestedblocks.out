PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ m
( ┃ (
) ┃ )
{ ┃ {
INT ┃ int
IDENTIFIER ┃ x
= ┃ =
INTEGER_LITERAL ┃ 42
; ┃ ;
INT ┃ int
IDENTIFIER ┃ y
= ┃ =
INTEGER_LITERAL ┃ 87
; ┃ ;
{ ┃ {
INT ┃ int
IDENTIFIER ┃ z
= ┃ =
IDENTIFIER ┃ x
+ ┃ +
IDENTIFIER ┃ y
; ┃ ;
RETURN ┃ return
IDENTIFIER ┃ z
; ┃ ;
} ┃ }
} ┃ }
} ┃ }
